x-pack/legacy/plugins/apm/public/utils/formatters.ts

```diff
@@ -149,7 +149,7 @@ export function asTime(
   value: FormatterValue,
   { withUnit = true, defaultValue = NOT_AVAILABLE_LABEL }: FormatterOptions = {}
 ) {
-  if (value == null) {
+  if (value == null || value === undefined) {
```

Comment:
`value == null` is equivalent to `value === null || value === undefined`

Reply:
Ah, totally missed that this was `==` and not `===`. Not used to the former anymore 😅